src/boost_claims.ts

```diff
@@ -0,0 +1,72 @@
+import { Block, EventWithTransaction } from "./common/deps.ts";
+import {
+  formatFelt,
+  SELECTOR_KEYS,
+  BOOST_CONTRACT,
+  MONGO_CONNECTION_STRING,
+  ID_UPGRADE_A_BLOCK,
+  STARKNET_QUEST_MONGODB_CONNECTION_STRING,
+  FINALITY,
+  AR_FINALITY,
+} from "./common/constants.ts";
+import { hash, BN } from "./common/deps.ts";
+
+const filter = {
+  header: { weak: true },
+  events: [
+    {
+      fromAddress: formatFelt(BOOST_CONTRACT),
+      keys: [hash.getSelectorFromName("OnClaim")],
+    },
+  ],
+};
+
+export const config = {
+  streamUrl: Deno.env.get("STREAM_URL"),
+  startingBlock: Number(Deno.env.get("STARTING_BLOCK")),
+  network: "starknet",
+  filter,
+  sinkType: "mongo",
+  finality: FINALITY,
+  sinkOptions: {
+    connectionString: STARKNET_QUEST_MONGODB_CONNECTION_STRING,
+    database: "goerli",
```

Comment:
please use an ENV variable for this, so we can specify if it is goerli or mainnet